.github/workflows/build_cmake.yml

```diff
@@ -30,13 +31,8 @@ jobs:
         os: ['ubuntu-latest', 'windows-latest', 'macos-latest']
         include:
         - os: ubuntu-latest
-          ENABLE_COVERAGE: ON
         - os: windows-latest
-          # Windows works differently
-          ENABLE_COVERAGE: OFF
         - os: macos-latest
```

Comment:
Now that `ENABLE_COVERAGE` is `ON` for all platforms, is this `include` block necessary?

Reply:
No. We can remove it.